src/ralph/deployment/tests/test_transitions.py

```diff
@@ -0,0 +1,267 @@
+from django.contrib.auth import get_user_model
+from django.core.urlresolvers import reverse
+from django.test import Client
+from rest_framework.test import APIClient
+
+from ralph.assets.models import AssetLastHostname
+from ralph.data_center.models import DataCenterAsset
+from ralph.data_center.tests.factories import (
+    DataCenterAssetFactory,
+    RackFactory
+)
+from ralph.lib.transitions.conf import TRANSITION_ORIGINAL_STATUS
+from ralph.lib.transitions.tests import TransitionTestCase
+from ralph.networks.tests.factories import (
+    NetworkEnvironmentFactory,
+    NetworkFactory
+)
+# from ralph.virtual.models import VirtualServer
```

Comment:
for the same reason as below code - it was waiting for separated PR to be merged (#2481).